.github/actions/setup-k8s-charm/action.yml

```
@@ -112,3 +112,9 @@ runs:
     - name: Disable analytics
       run: juju config dashboard analytics-enabled=false
       shell: bash
+    - name: Clean up
+      run: |
+        # Delete build files to save space. Pass `|| true` so that an error is not thrown in the directories don't exist.
```

I'm guessing there is a typo somewhere here?? or maybe I'm wrong?
```suggestion
        # Delete build files to save space. Pass `|| true` so that an error is not thrown if the directories don't exist.
```

---

Oops! Thanks, fixed.